src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const todoList = [
+  {
+    id: "1",
+    title: "Complete Assignment",
+   },
+   {
+     id: "2",
+     title: "Read",
+   },
+   {
+     id: "3",
+     title: "Be active",
+   },
+];
+
+function App() {
+  return (
+    <div style={{ textAlign: 'left' }}>
+      <header>
```

Comment:
you don't need <header> and <p> elements here
try to refresh the knowledte of html elements and how to use them
refer to w3 wedsite https://www.w3schools.com/html/html_intro.asp